### ISSUE TITLE
Compose File & Mounting External Hard Drive

### ISSUE THREAD

**Issue by getinthespirits** [sebgl/htpc-download-box#56]
I already have a significant media library and want to be able to make this available to plex as well. I am aware that I likely need to change the mount point for the hard drive and to adjust the docker compose yaml file to pick the folders up as well. 

What would be the best way of going about this?

**Comment by emkowale** [comment]
Yeah this was a trip to configure. You have to wrap your brain around how the docker-compose.yml works. Look at my deluge section here. Notice the difference between my downloads folder and config file.

  deluge:
    container_name: deluge
    image: linuxserver/deluge:latest
    restart: unless-stopped
    network_mode: host
    environment:
      - PUID=${PUID} # default user id, defined in .env
      - PGID=${PGID} # default group id, defined in .env
      - TZ=${TZ} # timezone, defined in .env
    volumes:
      - /mnt/downloads:/downloads # downloads folder
      - /home/pi/config/deluge:/config # config files

My directory structure is like this:

/home/pi/config/(ALL OF THE CONFIG FILES)

/home/pi/htpc-download-box/(ALL OF THE STUFF TO BUILD THIS PACKAGE)

/mnt/downloads  -  This is a mount to a physical hard drive on another raspberry pi.
In the /etc/fstab file I do:
10.255.252.33:/mnt/drive2 /mnt/downloads nfs rw,nofail 0 0
That makes the mount happened on boot. Notice the nofail. Very cool. If the remote raspberry pi isn't available for whatever reason, your pi will still boot.

So...
/mnt/downloads/incomplete
/mnt/downloads/complete
/mnt/downloads/complete/tv
/mnt/downloads/complete/movies

That's my story. The only thing I don't have working is the renaming and removing. Sonarr and Radarr simply will not remove the old torrents from Deulge once they are done, but that's another issue I might post when I'm not at work.  :)

**Comment by emkowale** [comment]
Thank you my friend. I'll look at that right now.

**Comment by emkowale** [comment]
Bummer. Not it. I just checked and /mnt/downloads is owned by pi which is my $USER specified in the .env file. Also, when I mount the /mnt/downloads share using 10.255.252.33:/mnt/drive2 /mnt/downloads nfs rw,nofail 0 0 in the fstab file everything is owned by root with rwxrwxrwx.
